pkg/kube/client/client.go

```diff
@@ -19,36 +25,18 @@ func NewClient(c k8sClient.Client) Client {
 
 type Client interface {
```

Comment:
package client with a Client interface doesn't say anything
not sure it belongs to this PR, but this would need to be changed eventually

Reply:
sure, we can decide on something better. Currently this is just an extension of the operator-sdk `client.Client` but we can for sure come up with a better name 👍

Reply:
operatorsdkclient in any form and order would work ;)